lib/aika_web/controllers/user_controller.ex

```diff
@@ -68,6 +68,14 @@ defmodule AikaWeb.UserController do
     redirect(conn, to: user_path(conn, :index))
   end
 
+  def generate_api_token(conn, %{"id" => id}) do
+    # TODO
+    conn.assigns[:user]
+    |> User.api_token_changeset()
```

Comment:
Could this call to the schema be moved to the `Aika.Accounts` module which handles the entire Accounts context. The requested change to move the Repo.update out of the changeset function I made above would cause for us to need to call Repo in this controller as well.

The Accounts context already has cases for updating parts of User such as the set_admin action